website/docs/getting-started/setup/instance-management/update-appsmith.mdx

```diff
@@ -1,366 +1,126 @@
-
-## Kubernetes
-
-Follow the steps to update your Appsmith installation:
-
-<Tabs groupId="appsmithEditions" queryString="current-edition">
-
-<TabItem label="Community Edition" value="communityEdition"> 
-
-You can update Appsmith in one of the below ways:
-
-### Automatic update
-
-The default Appsmith helm installation disables auto update (_recommended_). You have the option to enable auto update by either setting it in the values file or using a command.
-
-* Running the command below.
-  ```bash 
-    helm install appsmith/appsmith appsmith \
-      #highlight-next-line
-      --set autoupdate.enabled=true
-  ```
-
-* Or edit `values.yaml` and specify the following parameters
-```bash
-helm upgrade -i appsmith appsmith/appsmith -f values.yaml
-```
-
-### Manual update
-
-To manually update the Appsmith container image to the latest release, run the command: 
-
-```bash
-kubectl rollout restart statefulset appsmith
-```
-
-</TabItem>
-
-<TabItem label="Business Edition" value="businessEditin">
-
-You can update Appsmith in one of the below ways:
-
-### Automatic update
-
-The default Appsmith helm installation disables auto update (_recommended_). You have the option to enable auto update by either:
-  - Running the command below.
-    ```bash
-    helm upgrade --set autoupdate.enabled=true \
-    appsmith appsmith-ee/appsmith 
-    ```
-  - Or edit `values.yaml` and specify the following parameters
-    ```yaml
-    autoupdate.enabled: true
-    autoupdate.scheduler: 0
-    ```
-    Followed by running the command below.
-    ```bash
-    helm upgrade -i appsmith appsmith-ee/appsmith -f values.yaml
-    ```
 
-### Manual update
+* At least 2 GB of free storage space for backup and update tasks.
 
-To manually update the Appsmith container image to the latest release, run the command: 
+## Before you begin
 
-```bash
-kubectl rollout restart deployment appsmith
-```
+* Create a backup of the Appsmith instance, See [Backup Appsmith Instance](/getting-started/setup/instance-management/appsmithctl/#backup-instance) guide. (_Recommended_)
+* Upgrade your embedded or external MongoDB server to MongoDB v5 or later. See the list of [compliant platforms](https://www.mongodb.com/docs/manual/administration/production-notes/#platform-support) and follow the steps to [Upgrade a Replica Set to 5.0](https://www.mongodb.com/docs/manual/release-notes/5.0-upgrade-replica-set/).
 
-</TabItem>
-</Tabs>
+## Upgrade to checkpoint version
 
-## AWS ECS EC2
-Follow the below steps using the Classic Amazon Web Console to manually update Appsmith:
+A checkpoint version is a milestone release that ensures a proper application of any necessary database schema changes, optimisations, or other updates to prevent compatibility issues or data inconsistencies. When performing a manual upgrade, you first need to upgrade to a checkpoint version if your Appsmith installation version is prior to `v1.9.2`. The available Appsmith checkpoint versions are as follows:
```

Comment:
```suggestion
A checkpoint version is a milestone release that ensures a proper application of any necessary database schema changes, optimisations, or other updates to prevent compatibility issues or data inconsistencies. When performing a manual update, you first need to upgrade to a checkpoint version if your Appsmith installation version is prior to `v1.9.2`. The available Appsmith checkpoint versions are as follows:
```